core/commands/object/object.go

```diff
@@ -403,7 +403,7 @@ Available templates:
 `,
 	},
 	Arguments: []cmds.Argument{
-		cmds.StringArg("template", false, false, "Template to use. Optional."),
+		cmds.StringArg("template", false, false, "Template to use.").Default("false"),
```

Comment:
no default on arguments. This argument doesnt even appear to have a default value in the first place, its optional